docs/tutorials/migrations/nvm.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Node Version Manager (nvm)
+description: Using Flox to replace nvm
+---
+
+# Node Version Manager (nvm) migration guide
+
+Flox is an environment and package manager that allows you install software from an extensive catalog into individual [environments][environment_concept]{:target="\_blank"}, each of which usually corresponds to a software project. Flox gives you the opportunity to simplify the development workflow used by your team, consolidating various functions into a single command: `flox activate`.
+
+Just like [nvm](https://github.com/nvm-sh/nvm){:target="\_blank"}, Flox allows you to install and activate different versions of Node.js, allowing developers to switch versions as project requirements dictate.
+
+## Why you might want to use Flox instead of nvm
+nvm does exactly what it purports to do: it manages Node.js versions simply and effectively. Point notwithstanding, it's also one more dependency that you don't have to worry about if you're using Flox. Consider whether one of the following cases applies to you, or to your team:
```

Comment:
```suggestion
`nvm` does exactly what it purports to do: it manages Node.js versions simply and effectively. Point notwithstanding, it's also one more dependency that you don't have to worry about if you're using Flox. Consider whether one of the following cases applies to you, or to your team:
```

Reply:
I'm not so sure about this. I would use the backticks if I were using `nvm` as a command (e.g., `nvm use`), but here, I'm referring to nvm the "product." Their own README is inconsistent on this, but more often than not, they use the term "nvm" without formatting it as code. Additionally, it's always lowercase (unless it's included as a part of some environment variable). It may seem that I was being sloppy here, but this is actually quite an intentional usage.

Reply:
Agree. In sentence usage we should talk about the tool nvm without backticks.